lib/rust/frp/src/io/js.rs

```diff
@@ -91,82 +91,42 @@ fn event_listener_options() -> enso_web::AddEventListenerOptions {
 /// `pass_to_dom` event will be emitted.
 ///
 /// To make this class manage js event, you should wrap the closure passed as event listener using
-/// `make_event_handler` function.
+/// `handler` function.
```

Comment:
Update the docs. We do not have "pass_to_dom" anymore.

In fact, I'm not sure if we need this function anymore. Before, we wanted to keep `JsEvent` to be able to call "prevent_default" if nobody allowed passing it to DOM. Now, I don't see any scenario where we need global access to currently processed event.

Reply:
I'm not sure here. I like this structure, as it wraps all JS events in FRP. I'd leave it this way, as it should not affect performance (we are doing that only once when registering bindings) and it gives nice FRP access to all JS events. Seems handy. Docs updated.